packages/gradient/src/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { LinearGradient } from 'expo-linear-gradient'
+import { styled, useDripsyTheme } from '@dripsy/core'
+
+type Props = Omit<React.ComponentProps<typeof LinearGradient>, 'colors'> & {
+  gradient?: string
+  colors?: string[]
+}
+
+const Grad = styled(
+  React.memo(
+    function Gradient(props: Props) {
+      const { colors, linearGradients } = useDripsyTheme().theme
+      // Helper function to map colors to appropriate theme colors
+      const colorArrayToTheme = (colorArray: typeof props.colors) => {
+        // Return an empty array if the colors come back as undefined
+        return (
+          colorArray?.map((color) => (colors?.[color] as string) ?? color) ?? []
+        )
+      }
+      let gradientColors: string[] = []
+      if (props.gradient) {
```

Comment:
We should maybe add a check / warn a user if `linearGradients?.[props.gradient]` doesn't exist.